src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -336,9 +336,12 @@ def check_created_partition_table(targetMap, installation)
       if (!boot_found && installation && 
           !Partitions.EfiBoot && rootdlabel=="gpt") || show_all_popups
         message = _(
-          "Warning: There is no partition of type bios_grub present.\n" +
-          "To boot from a GPT disk using grub2 such a partition is needed.\n" +
-          "\n" +
+          "Warning: There is no partition of type bios_grub present.\n" \
+          "When installing Grub2 to the MBR of a GPT formatted disc it is recommended\n" \
```

Comment:
@aschnell, @kobliha You are reviewing the original text that was already agreed to be... let's say... suboptimal. Please review the one proposed in https://github.com/yast/yast-storage/pull/137#discussion_r32722471